.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,6 +9,21 @@ https://git.k8s.io/community/contributors/devel/sig-release/release.md#issuepr-k
 6. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
 -->
 
+**Which is the sponsoring or primary SIG associated with this PR?**
+<!--
+Label the sponsoring or primary SIG associated with this PR.
+
+For reference on available SIGs, you can find more details at:
+https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md#labels-that-apply-to-all-repos-for-both-issues-and-prs
```

Comment:
This link doesn't really tell you what to do; it's a list of labels. It doesn't say how to add a label to a PR (the bot commands are subtly different from the list of labels).
I'm actually confused if this wants me to _label_ the SIG, or write a SIG name in the block, or both? And since formatting matters in both cases, we'd want an example for each.